test/ReadsUtils_server_test.py

```diff
@@ -1207,7 +1207,11 @@ def check_lib(self, lib, size, filename, md5):
         file_md5 = hashlib.md5(open(path, 'rb').read()).hexdigest()
         libfile = lib['file']
         self.assertEqual(file_md5, md5)
-        self.assertEqual(lib['size'], size)
+        size_buffer = 0.15  # allow +-15% size to permit a chagne in compression algorithm
+        min_size = int(float(size) * size_buffer - float(size))
```

Comment:
This is a negative number

Reply:
If the thing is supposed to be +-15% then maybe instead:
```p
min_size = int(float(size) * 0.85)
max_size = int(float(size) * 1.15)
```